<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeftDockWidget</class>
 <widget class="QDockWidget" name="LeftDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>515</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>图层控制</string>
  </property>
  <widget class="QWidget" name="widget">
   <property name="styleSheet">
    <string notr="true">QPushButton{	border: none;color: #000000;}
QPushButton:hover{border: 1px solid #E0B66E;}
QPushButton:pressed{border: 1px solid #DE951C;border-bottom: none;color: white;}
QPushButton:checked{border: 1px solid #DE951C;border-bottom: none;color: white;}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="mainwindow.qrc">
          <normaloff>:/images/mapToolBar/top.png</normaloff>:/images/mapToolBar/top.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="mainwindow.qrc">
          <normaloff>:/images/mapToolBar/bottom.png</normaloff>:/images/mapToolBar/bottom.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_addbaseMap">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="mainwindow.qrc">
          <normaloff>:/images/defaultToolBar/map.png</normaloff>:/images/defaultToolBar/map.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
